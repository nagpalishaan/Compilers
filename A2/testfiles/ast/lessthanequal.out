Program
  List
    Function
      IdDecl ID="n"
      List
        DeclareVar
          IdDecl ID="a"
        DeclareVar
          IdDecl ID="b"
        AssignVar
          IdDecl ID="c"
          LessEqual
            IdUse ID="a"
            IdUse ID="b"
