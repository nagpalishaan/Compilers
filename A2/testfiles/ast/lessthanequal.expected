Program
  List
    Function
      IdDecl ID="n"
      List
      List
        DeclareVar
          IdDecl ID="a"
          Opt
        DeclareVar
          IdDecl ID="b"
          Opt
        AssignVar
          IdDecl ID="c"
          LessEqual
            IdUse ID="a"
            IdUse ID="b"
